features/0511-dif-cred-manifest-attach/README.md

```diff
@@ -0,0 +1,292 @@
+*Credental Manifest* describes a data format that specifies the inputs an Issuer requires for issuance of a credential. It relies on the closely-related
+[*Presentation Exchange specification*](https://identity.foundation/presentation-exchange/) to describe the required inputs and
+the format in which the Holder submits those inputs (a verifiable presentation).
+
+
+## Motivation
+
+The *Credential Manifest* specification lends itself well to several transport mediums due to its limited scope as a data format,
+and is easily transported over DIDComm.
+
+It is furthermore desirable to make use of specifications developed in an open standards body.
+
+
+## Tutorial
+
+Complete examples of messages are provided in the [reference section](#reference).
+
+Credential Manifests MAY be acquired by the Holder via out of band means, such as from a well-known location on the Issuer's website.
+This allows the Holder to initiate the `issue-credential` protocol with a `request-message` providing they also possess the requisite
+`challenge` and `domain` values. If they do not possess these values then the Issuer MAY respond with an `offer-credential` message.
+
+Otherwise the Holder MAY initiate the protocol with `propose-credential` in order to discover the Issuer's requirements.
+
+
+## Reference
+
+### `propose-credential` attachment format
+
+Format identifier: `dif/credential-manifest@v1.0`
+
+The contents of the attachment is the minimal form of the Issuer's credential manifest describing the credential the Holder desires.
+It SHOULD contain the `issuer` and `credential` properties and no more.
+
+Complete message example:
+
+```json
+{
+    "@id": "8639505e-4ec5-41b9-bb31-ac6a7b800fe7",
+    "@type": "https://didcomm.org/issue-credential/%VER/propose-credential",
+    "comment": "<some comment>",
+    "credential_proposal": <json-ld object>,
+    "formats" : [{
+        "attach_id": "b45ca1bc-5b3c-4672-a300-84ddf6fbbaea",
+        "format": "dif/credential-manifest@v1.0"
+    }],
+    "filter~attach": [{
+        "@id": "b45ca1bc-5b3c-4672-a300-84ddf6fbbaea",
+        "mime-type": "application/json",
+        "data": {
+            "json": {
+                "issuer": "did:example:123",
+                "credential": {
+                    "name": "Washington State Class A Commercial Driver License",
+                    "schema": "ipfs:QmPXME1oRtoT627YKaDPDQ3PwA8tdP9rWuAAweLzqSwAWT"
+                }
+            }
+        }
+    }]
+}
+```
+
+
+### `offer-credential` attachment format
+
+Format identifier: `dif/credential-manifest@v1.0`
+
+The contents of the attachment is a JSON object containing the Issuer's credential manifest, a challenge and domain:
+
+```jsonc
+{
+    "challenge": "...",
+    "domain": "...",
+    "credential_manifest": {
+        // credential manifest object
+    }
+}
+```
+
+Complete message example:
```

Comment:
I'm thinking a better approach would be to have a pointer to examples in the DIF document vs. putting examples inline here.  I think this document should provide context between the Issue Credential protocol and the DIF data model - how one is used with the other.

Reply:
I think an intentionally incomplete example is useful. Show some outer structure, and indicate the incompleteness with a comment.